InnerEye/ML/run_ml.py

```diff
@@ -223,6 +224,16 @@ def setup(self, use_mount_or_download_dataset: bool = True) -> None:
                                                     run_context=RUN_CONTEXT)
         self.checkpoint_handler.download_recovery_checkpoints_or_weights(only_return_path=not is_global_rank_zero())
 
+        if self.azure_config.pretraining_run_recovery_id is not None:
+            run_to_recover = self.azure_config.fetch_run(self.azure_config.pretraining_run_recovery_id.strip())
+            run_recovery_object = RunRecovery.download_all_checkpoints_from_run(self.container,
+                                                                                run_to_recover,
+                                                                                EXTRA_RUN_SUBFOLDER,
+                                                                                only_return_path=not is_global_rank_zero())
+            self.container.extra_downloaded_run_id = run_recovery_object
```

Comment:
This field "extra_downloaded_run_id" could do with some documentation. Also, we define that twice in DeepLearningConfig and in the container. Maybe it is better places in WorkflowParams?

Reply:
I've renamed `extra_downloaded_run_id` to `pretraining_run_checkpoints` and initialized it once in WorkflowParams. I've also moved `pretraining_run_recovery_id` from AzureConfig to WorkflowParams.

Reply:
Moving `pretraining_run_checkpoints` to WorkflowParams causes issues with initialization, I'm reverting this change for now.